samples/angular-todo/src/app/features/journey/form/form.component.ts

```diff
@@ -161,7 +199,17 @@ export class FormComponent implements OnInit {
    */
   handleStep(step?: FRStep) {
     this.step = step;
-
+    const selectIdCallback = step.callbacks.find((cb) => cb.payload.type === 'SelectIdPCallback');
+    const callBackInputValue = selectIdCallback?.payload.input[0].value as string;
+    if (selectIdCallback && callBackInputValue === '') {
+      selectIdCallback.payload.input[0].value = 'localAuthentication';
+    }
```

Comment:
Are we defaulting to `localAuthentication` here? If we are defaulting, do we also need the code above to essentially do the same thing? Or, if the above exists, does the code here need to exist? Maybe I'm misunderstanding the intention here, which is why I'd like to see comments, but it seems a bit redundant.

Reply:
removed